wchat/screens/ChatList/index.js

```diff
@@ -10,7 +10,7 @@ class ChatListContainer extends Component {
     this.props.dispatch(chatsActions.fetchChats())
   }
   onSelect = ({id}) => {
-    this.props.navigation.navigate('Chat', {id: id })
+    this.props.navigation.navigate('Chat', { id })
   }
   onAdd = () => {
```

Comment:
let the empty function and remove console.log()
`onAdd = () => {};`

One more thing. When declare a function into a container/smart class, call it `handleFunction` (in this case, `handleAdd`) and then pass this function like `onFunction` (in this case 
```
 <ContactTable
         data={this.props.chats}		       
         addIcon={add_chat_icon}		       
         onSelect={this.onSelect}		        
         onAdd={this.handleAdd}		       
         />		      
```
)
So its more descriptive taht this function is the handler taht you pass to children and the children ejecute it `on` an action